server_test.go

```diff
@@ -167,6 +198,153 @@ func TestLocalServer(t *testing.T) {
 	server.Flush(config.Interval, config.FlushLimit)
 }
 
+func TestLocalServerMixedMetrics(t *testing.T) {
+
+	// Same as in TestLocalServerUnaggregatedMetrics
+	var RemoteResponses int
+	defer func() {
+		assert.Equal(t, 1, RemoteResponses, "Server did not complete all responses before test terminated!")
+	}()
+
+	// Unlike flushing to to the remote server, flushing forward
+	// (ie, flushing to global veneur) is asynchronous.
+	// This means it is quite likely that the function will return before
+	// the global server has actually responded, unless we introduce a small timeout
+	GlobalResponseChan := make(chan struct{})
+	defer func() {
+		select {
+		case <-GlobalResponseChan:
+			// all is safe
+		case <-time.After(DefaultServerTimeout):
+			assert.Fail(t, "Global server did not complete all responses before test terminated!")
+		}
+	}()
+
+	// We can't be guaranteed that we will receive this exact stream
+	// but we should expect that they represent the same underlying tdigest
```

Comment:
could you clarify this comment? if i'm reading it correctly, it means "the t-digest that we decode might not have the same gob representation as this, but its min, max and count should be approximately the same", but it took a few readings to really understand